include/envoy/router/router.h

```diff
@@ -370,6 +370,22 @@ class ShadowPolicy {
 
 using ShadowPolicyPtr = std::unique_ptr<ShadowPolicy>;
 
+/**
+ * All virtual cluster stats. @see stats_macro.h
+ */
+#define ALL_VIRTUAL_CLUSTER_STATS(COUNTER)                                                         \
+  COUNTER(upstream_rq_retry)                                                                       \
```

Comment:
should we have tests for the other counters? I just see tests for 2.

Reply:
They all have tests:
1.`upstream_rq_retry` and `upstream_rq_retry_success`: https://github.com/envoyproxy/envoy/pull/10474/files#diff-0a6a6327b31a217b22c653b1d1ebf4acR829
2.`upstream_rq_timeout`: https://github.com/envoyproxy/envoy/pull/10474/files#diff-6eb802c7b8ea53fcfa4c3f2842b28c7bR1565
3.`upstream_rq_total`: all over router_test.cc

Reply:
Ah maybe router_test was collapsed in your view?

Reply:
Exactly "large diffs are hidden by default" bit me again. Sorry about the confusion!